src/inspector/view/qml/MuseScore/Inspector/notation/lines/internal/OttavaStyleSettings.qml

```diff
@@ -21,6 +21,8 @@
  */
 import QtQuick 2.15
 import QtQuick.Controls 2.15
+import QtQuick 2.3
```

Comment:
Do we need both QtQuick?

Reply:
yes

Reply:
Well, I believe the later version includes the older?

Reply:
yes

Reply:
So only the latter is needed.